src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -75,6 +79,24 @@ bool GetEntrypointExecutableAbsolutePath(std::string& entrypointExecutable)
             result = true;
         }
     }
+#elif defined (__FreeBSD__)
+    static const int name[] = {
+        CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, -1
+    };
+    char path[PATH_MAX];
+    size_t len;
+
+    len = sizeof(path);
+    if (sysctl(name, 4, path, &len, NULL, 0) == 0)
```

Comment:
Minor nit: can you use `nullptr` here to match the rest of the file (there is one other occurrence of `NULL` but the rest is all `nullptr`?